web/src/components/Note.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+
+import { NoteIcon } from '../assets'
+import Icon from './base/Icon'
+
+const NoteContainer = styled.div<{ visible: boolean }>`
+  display: flex;
+  background-color: ${props => props.theme.colors.themeColor};
```

Comment:
please use a separate `warningColor` instead of theme

Reply:
Created #2577.